homework-submission/react-hyf/server/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "Express server of react-hyf",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "server": "nodemon server.js",
+    "client": "cd .. && cd react-hyf-client && npm start",
```

instead of making three sentences we could make it two:
```suggestion
    "client": "cd ../react-hyf-client && npm start",
```

---

Of course this is better ... Thanks for this advice